README.md

```diff
@@ -76,34 +76,32 @@ Once you have this configuration, you can use the Google Cloud SDK to deploy thi
-|`HEAP_SIZE_MB`    | Available heap      | size     | 80% of `${GAE_MEMORY_MB}`                   |
-|`JAVA_HEAP_OPTS`  | JVM heap args       | JVM args | `-Xms${HEAP_SIZE_MB}M -Xmx${HEAP_SIZE_MB}M` |
-|`JAVA_GC_OPTS`    | JVM GC args         | JVM args | `-XX:+UseG1GC` plus configuration           |
-|`JAVA_GC_LOG`     | JVM GC log file     | filename |                                             |
-|`JAVA_GC_LOG_OPTS`| JVM GC args         | JVM args | Derived from `$JAVA_GC_LOG`                 |
-|`JAVA_USER_OPTS`  | JVM other args      | JVM args |                                             |
-|`JAVA_OPTS`       | JVM args            | JVM args | See below                                   |
-
-If not explicitly set, `JAVA_OPTS` is defaulted to 
-```
-JAVA_OPTS:=-showversion \
-           ${JAVA_TMP_OPTS} \
-           ${DBG_AGENT} \
-           ${JAVA_HEAP_OPTS} \
-           ${JAVA_GC_OPTS} \
-           ${JAVA_GC_LOG_OPTS} \
-           ${JAVA_USER_OPTS}
-```
+
+The [/docker-entrypoint.bash](https://github.com/GoogleCloudPlatform/openjdk-runtime/blob/master/openjdk8/src/main/docker/docker-entrypoint.bash)
+for the image is inherited from the openjdk-runtime and it's capabilities are described in the associated 
+[README](https://github.com/GoogleCloudPlatform/openjdk-runtime/blob/master/README.md)
+
+This image updates the docker `CMD` and appends to the
+[setup-env.bash](https://github.com/GoogleCloudPlatform/openjdk-runtime/blob/master/openjdk8/src/main/docker/setup-env.bash)
+script to include options and arguments to run the Jetty container, unless an executable argument is passed to the docker image.
+Addition environment variables are set including:
+
+|Env Var           | Maven Prop      | Value                                                |
+|------------------|-----------------|------------------------------------------------------|
+|`JETTY_VERSION`   |`jetty9.version` |                                                      |
+|`GAE_IMAGE_NAME`  |                 |`jetty`                                               |
+|`GAE_IMAGE_LABEL` |`docker.tag.long`|                                                      |
+|`JETTY_HOME`      |`jetty.home`     |`/opt/jetty-home`                                     |
+|`JETTY_BASE`      |`jetty.base`     |`/var/lib/jetty`                                      |
+|`TMPDIR`          |                 |`/tmp/jetty                                           |
+|`JETTY_ARGS`      |                 |`-Djetty.base=$JETTY_BASE -jar $JETTY_HOME/start.jar` |
+|`ROOT_WAR`        |                 |`$JETTY_BASE/webapps/root.war`                        |
+|`ROOT_DIR`        |                 |`$JETTY_BASE/webapps/root`                            |
+|`JAVA_OPTS`       |                 |`$JAVA_OPTS $JETTY_ARGS`                              |
+
+If a WAR file is found at `$ROOT_WAR`, it is unpacked to `$ROOT_DIR` if it is newer than the directory or the directory
```

Comment:
I think this is a bit confusing. Maybe adding an example of a Dockerfile that includes a WAR and an example that includes an exploded WAR would help.